docs/node/admin/kwil-admin/snapshot/create.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+sidebar_label: "create"
+id: "kwil-admin-snapshot-create"
+title: "kwil-admin snapshot create"
+description: "Creates a snapshot of the database."
+slug: /ref/kwil-admin/snapshot/create
+---
+
+## kwil-admin snapshot create
+
+Creates a snapshot of the database.
+
+### Synopsis
+
+
+This command creates a logical snapshot of the database and genesis file to initialize a new network. The genesis file is created with the app_hash set to the hash of the snapshot(uncompressed) and the genesis validators set to the validators from the previous network. This command interacts directly with the underlying PostgreSQL server, bypassing any interactions with the kwild node. It requires user with administrative privileges on the database to create snapshot without any restrictions. 
```

Comment:
`snapshot(uncompressed)` => `snapshot (uncompressed)`

although I think the uncompressed part is irrelevant to the command's user.  This command creates a logical snapshot and prepares a genesis.json for a new network based on that snapshot.  Any deeper is implementation detail unless it affect the user's actions.

---

"It requires user with administrative privileges on the database to create snapshot without any restrictions."

=>

"It requires a database user with superuser privileges."


---

If referring field of genesis.json, should backtick like `app_hash`.

---